src/python/pants/backend/python/python_artifact.py

```diff
@@ -71,10 +71,11 @@ def __str__(self):
     return self.name
 
   def _compute_fingerprint(self):
-    fingerprint = sha1(json.dumps((self._kw, self._binaries),
+    json_representation = json.dumps((self._kw, self._binaries),
                                   ensure_ascii=True,
                                   allow_nan=False,
-                                  sort_keys=True)).hexdigest()
+                                  sort_keys=True)
+    fingerprint = sha1(json_representation.encode('utf-8')).hexdigest()
```

Comment:
Is there a reason we're not using `pants.base.hash_utils.stable_json_hash()`?

Reply:
No good reason at all beyond using what already existed in this file. Good suggestion!